tests/app/questionnaire/test_path_finder.py

```diff
@@ -414,12 +414,17 @@ def test_remove_answer_and_block_if_routing_backwards():
                 "section_id": "default-section",
                 "list_item_id": None,
                 "status": CompletionStatus.COMPLETED,
+                "block_ids": ["route-backwards-block"],
```

Comment:
Looks fine. Since this issue wasn't picked up by our existing tests shouldn’t we add some functional one to be safer in the future?

Reply:
See the PR description around the test. This same test would fall on `main` because the routing path would be of different size.